### ISSUE TITLE
Confirm the XL/XXL upper end of the limit on buckets are calculated correctly.

### ISSUE THREAD

**Issue by itsacoyote** [itsacoyote/split-ting#4]
The bucket limits are calculated with the 5% grace on all plans beyond XS. However I noticed I left, for example, the Minutes XL bucket tops at 2100 as listed on the Ting Rates page. I believe I forgot to add in the 5% to that top limit on the Minutes, Megabytes and Messages. Will likely need to be updated on the old and new rates.

**Comment by TPS** [comment]
@itsacoyote [Reportly](https://help.ting.com/hc/en-us/community/posts/203757888/comments/223878487?community_id=200115358#community_comment_223878487), [the new data rates aren't accounted for](https://ting.com/blog/ting-drops-data-prices-to-10-gb-beyond-the-first-gig/), either.

**Comment by itsacoyote** [comment]
@TPS Yeah, it's been a while since the last commit, just got too busy and didn't get back to updating it. I'm looking into rewriting the whole thing and improving it and getting it up to date with the new rate system.
